org.siril.Siril.json

```diff
@@ -41,6 +44,110 @@
                 }
             ]
         },
+        {
+            "name": "gtksourceview4",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://deb.debian.org/debian/pool/main/g/gtksourceview4/gtksourceview4_4.8.4.orig.tar.xz",
```

Comment:
you should avoid using debian archives.

upstream is here: https://download.gnome.org/sources/gtksourceview/4.8/